jake/day2/solutions/part2.mjs

```diff
@@ -0,0 +1,90 @@
+console.time('runTime');
+import { once } from 'events';
+import { createReadStream } from 'fs';
+import * as readline from 'node:readline/promises';
+
+const PLAY = {
+  DRAW: 3,
+  WIN: 6,
+  LOSS: 0,
+  ROCK: 1,
+  ROCK_WIN: 'SCISSORS',
+  ROCK_LOSS: 'PAPER',
+  PAPER: 2,
+  PAPER_WIN: 'ROCK',
+  PAPER_LOSS: 'SCISSORS',
+  SCISSORS: 3,
+  SCISSORS_WIN: 'PAPER',
+  SCISSORS_LOSS: 'ROCK',
+  A: 'ROCK',
+  B: 'PAPER',
+  C: 'SCISSORS',
+  X: 'LOSS',
```

Comment:
I like this, it took some brain gymnastics the way I did it to keep things straight.